modules/gltf/gltf_document.cpp

```diff
@@ -4803,9 +4803,10 @@ Error GLTFDocument::_parse_animations(Ref<GLTFState> state) {
 		Array samplers = d["samplers"];
 
 		if (d.has("name")) {
-			const String name = d["name"];
-			if (name.begins_with("loop") || name.ends_with("loop") || name.begins_with("cycle") || name.ends_with("cycle")) {
+			String name = d["name"];
+			if (name.begins_with("loop") || name.ends_with("loop") || name.begins_with("cycle") || name.ends_with("cycle") || name.find(" -loop") > 0) {
```

Comment:
Is the space before the `-` intended? I can't see why you'd name an animation `some_anim -loop` instead of `some_anim-loop`.

Reply:
I think it should be without space to be consistence with other type or naming like `-noimp` which works without space

Reply:
iirc meshes the arguments are like `box -colonly` this is why i included the space